keynote mario munoz (#54)
djangocon/2024.djangocon.us: src/_content/posts/introducing-keynote-mario-munoz.md

---
author: DjangoCon US Communications Team
category: General

> date: 2024-08-01 06:00:00

Reviewer: ```suggestion
date: 2024-08-01 06:00:00-04:00
```

cover:
    url: /assets/img/blog/mario_munoz.jpg
    alt: "Mario Munoz"
layout: post
title: "Announcing Our Keynotes: Mario Munoz"
---

> ### We're honored to announce our seecond keynote speaker for DjangoCon US 2024: Mario Munoz!

Reviewer 2: @jcjudkins is there a reason why this is a `h3` element instead of `h2`?

```suggestion
### We're honored to announce our second keynote speaker for DjangoCon US 2024: Mario Munoz!
```

> We had the opportunity to sit down with Mario and ask her a few questions about his work, his upcoming talk, and what shehee's looking forward to at DjangoCon US 2024.

Reviewer 2: ```suggestion
We had the opportunity to sit down with Mario and ask him a few questions about his work, his upcoming talk, and what he's looking forward to at DjangoCon US 2024.
```

Reviewer: jinx

Reviewer 2: Yeah, but I bet you didn't first change `her` to `his`!

Reviewer: you are indeed correct

> We had the opportunity to sit down with Mario and ask her a few questions about his work, his upcoming talk, and what shehee's looking forward to at DjangoCon US 2024.

Reviewer: ```suggestion
We had the opportunity to sit down with Mario and ask him a few questions about his work, his upcoming talk, and what he's looking forward to at DjangoCon US 2024.
```

> ## Can you tell us how you use Django

Reviewer: ```suggestion
## Can you tell us how you use Django?
```

> ## Can you tell us how you use Django

Reviewer 2: ```suggestion
## Can you tell us how you use Django?
```

> >When preparing my first talk on "why I didn't choose Django," I thought I should at least build a Django app before getting into it... As a result, I built an app to hold my presentation slides (also using reveal.js), and ever since, I've continued using it for the past two years.

Reviewer 2: ```suggestion
>When preparing my first talk on ["Why I Didn't Start With Django"](https://2022.djangocon.us/talks/why-i-didn-t-start-with-django/), I thought I should at least build a Django app before getting into it... As a result, I built an app to hold my presentation slides (also using reveal.js), and ever since, I've continued using it for the past two years.
```

## Can you give us a teaser about what your talk is about?

> >I am very interested in the intersection of people and technology. What role does community have in our technological journey? How do we see ourselves within the context of a "technical" community?  At a recent conference (North Bay Python), I spoke about building bridges between communities. My talk at DjangoCon will be an extension of this. My current working title is "How to be a developer, and other lies we tell ourselves." We'll be going through a journey that brings us in and out of different mindsets as developers, and what role community plays in all of this. As many of my talks, I often have many biographical/personal/meta examples...

Reviewer 2: ```suggestion
>I am very interested in the intersection of people and technology. What role does community have in our technological journey? How do we see ourselves within the context of a "technical" community?  At a recent conference (North Bay Python), I spoke about building bridges between communities. My talk at DjangoCon US will be an extension of this. My current working title is "How to be a developer, and other lies we tell ourselves." We'll be going through a journey that brings us in and out of different mindsets as developers, and what role community plays in all of this. As many of my talks, I often have many biographical/personal/meta examples...
```

> >I am very interested in the intersection of people and technology. What role does community have in our technological journey? How do we see ourselves within the context of a "technical" community?  At a recent conference (North Bay Python), I spoke about building bridges between communities. My talk at DjangoCon will be an extension of this. My current working title is "How to be a developer, and other lies we tell ourselves." We'll be going through a journey that brings us in and out of different mindsets as developers, and what role community plays in all of this. As many of my talks, I often have many biographical/personal/meta examples...

Reviewer: ```suggestion
>I am very interested in the intersection of people and technology. What role does community have in our technological journey? How do we see ourselves within the context of a "technical" community?  At a recent conference (North Bay Python), I spoke about building bridges between communities. My talk at DjangoCon US will be an extension of this. My current working title is "How to be a developer, and other lies we tell ourselves." We'll be going through a journey that brings us in and out of different mindsets as developers, and what role community plays in all of this. As many of my talks, I often have many biographical/personal/meta examples...
```

Should we link this NBPy talk?

> >I am very interested in the intersection of people and technology. What role does community have in our technological journey? How do we see ourselves within the context of a "technical" community?  At a recent conference (North Bay Python), I spoke about building bridges between communities. My talk at DjangoCon US will be an extension of this. My current working title is "How to be a developer, and other lies we tell ourselves." We'll be going through a journey that brings us in and out of different mindsets as developers, and what role community plays in all of this. As many of my talks, I often have many biographical/personal/meta examples...

Reviewer 2: ```suggestion
>I am very interested in the intersection of people and technology. What role does community have in our technological journey? How do we see ourselves within the context of a "technical" community?  At a recent conference (North Bay Python), [I spoke about building bridges between communities](https://pretalx.northbaypython.org/nbpy-2024/talk/X7ZZPP/). My talk at DjangoCon US will be an extension of this. My current working title is "How to be a developer, and other lies we tell ourselves." We'll be going through a journey that brings us in and out of different mindsets as developers, and what role community plays in all of this. As many of my talks, I often have many biographical/personal/meta examples...
```

## What are you most looking forward to at DjangoCon US 2024?

> >I am most excited about engaging with the community. I am currently involved in the Djangonaut program and look forward to seeing fellow Djangonauts there. I've also seen that the talks have been posted, and some of them look fantastic. I'm interested in all things hypermedia, so definitely will be catching those htmx-related talks.

Reviewer: ```suggestion
>I am most excited about engaging with the community. I am currently involved in the Djangonaut Space program and look forward to seeing fellow Djangonauts there. I've also seen that the talks have been posted, and some of them look fantastic. I'm interested in all things hypermedia, so definitely will be catching those HTMX-related talks.
```

## What are your favorite topics at conferences?

> >As mentioned, I'm excited about the hypermedia. But I always like hearing about broad topics, such as the past, present, and future of Django talk by Will, or Jacob's talk on what the DSF could accomplish with 4x the budget.

Reviewer: It would be useful to link to those talks

Reviewer 2: I didn't link to them immediately because I didn't want to hard-code them and I didn't know the e11nty way to do that yet.

Reviewer: asked about it in slack

> >As mentioned, I'm excited about the hypermedia. But I always like hearing about broad topics, such as the past, present, and future of Django talk by Will, or Jacob's talk on what the DSF could accomplish with 4x the budget.

Reviewer 2: ```suggestion
>As mentioned, I'm excited about the hypermedia. But I always like hearing about broad topics, such as [the past, present, and future of Django's User Model talk by Will](/talks/django-user-model-past-present-and-future/), or [Jacob's talk on what the DSF could accomplish with 4x the budget](/talks/django-user-model-past-present-and-future/).
```

## How do you view the connections made at conferences?

> The connections we make are invaluable. As an introvert, like many of us, I seek meaningful and strong connections. The Django community, in particular, has been a welcoming space for these interactions.

## Can you tell us about your career journey?

> > I started my career in payroll and then moved to an HR analyst role, providing IT-related support. It was through networking that I met people who helped me transition into engineering. I met some key contacts at PyCon.

Reviewer: ```suggestion
> I started my career in payroll and then moved to an HR analyst role, providing IT-related support. It was through networking that I met people who helped me transition into engineering. I met some key contacts at PyCon US.
```

I'm assuming here

## How did you approach your career growth?

> I purposely leaned into areas where I wasn’t as strong, getting out there and trying to speak. It’s a long-term investment, and it takes time to get there. Purposefully plugging into the community and focusing on what you can do with code, whether you're a newcomer or experienced, is crucial. Learning not to gatekeep is also important.

## When did you start speaking at conferences, and what has your experience been like?

> > I started speaking in 2022, giving talks at PyCon and PyOhio. I've done tutorials and talks, and it's been an incredible experience that has helped me grow both personally and professionally.

Reviewer: ```suggestion
> I started speaking in 2022, giving talks at PyCon US and PyOhio. I've done tutorials and talks, and it's been an incredible experience that has helped me grow both personally and professionally.
```

## What do you like to do outside of your normal work day?

> >Beach days with the family are always fun, or adventuring in a new/unknown hiking area. And, well, I have a blog called Python By Night, so that kind of speaks for itself.

Reviewer 2: ```suggestion
>Beach days with the family are always fun, or adventuring in a new/unknown hiking area. And, well, I have a blog called [Python By Night](https://pythonbynight.com/), so that kind of speaks for itself.
```

> >Beach days with the family are always fun, or adventuring in a new/unknown hiking area. And, well, I have a blog called Python By Night, so that kind of speaks for itself.

Reviewer: We should link to it  (python by night dot com?)
